src/components/PatientRecord/PatientRecord.tsx

```diff
@@ -102,8 +106,9 @@ const PatientRecordElement: FC<PatientRecordElementProps> = ({ resourceType }) =
   const [isExpanded, setIsExpanded] = useState<boolean>(false);
 
   const chevron: IconProp = isExpanded ? faChevronUp : faChevronDown;
-  const resourceCount: string =
-    resourceType !== 'Patient' ? '(' + resourcesByType.length + ')' : '';
+  const resourceCount: string = !['Patient', 'Pathway'].includes(resourceType)
+    ? '(' + resourcesByType.length + ')'
```

Comment:
prefer template literal
```suggestion
    ? `(${resourcesByType.length})`
```